andela/dahlia-ah-backend
# #167164980 send verification email

#### Pivotal tracker story
[#167164980](https://www.pivotaltracker.com/story/show/167164980)
#### What does this PR do?
- verifies user 
#### Summary of Task
- create a function to send an email verification link
- create email verification endpoint 
#### How can this be tested?
- clone repo to local computer run npm script ```npm run db:ready``` and  ```npm run dev:start```
- on post man sign up using the endpoint ```POST api/v1/users``` you will receive an email after entering your details
- Click on the link in the email and check the isVerified field for the created user
#### Screenshots (if appropriate)
